argo-apps/tooling-app-of-apps/values.yaml

```diff
@@ -26,5 +26,7 @@ applications:
      path: camelk-community  
    - name: tekton
      path: tekton  
-#    - name: devspaces
-#      path: devspaces  
+   - name: devspaces
+     path: devspaces  
```

Comment:
Also, dev spaces should be upgraded to 3.3 version which was released recently

Reply:
also done